content/docs/addons-test-utils.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: test-utils
-title: Test Utilities
+title: Utilitats de Test 
 permalink: docs/test-utils.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**Fent la Importació**
```

Comment:
```suggestion
**Important**
```

Reply:
As you know, I appreciate so much, your suggested changes. But in this case I have to disagree. In our language "important" is translated to english as important, and this is not the sense of the phrase.

Here  ***importing*** is more convenient to translate it  as "**importació**"